package.json

```
@@ -11,8 +11,9 @@
   },
   "main": "./dist/node/scratch-storage.js",
   "browser": "./dist/web/scratch-storage.js",
+  "types": "./dist/types/index.d.ts",
   "scripts": {
-    "build": "webpack --progress --colors --bail",
+    "build": "rm -rf dist && webpack",
```

```suggestion
    "build": "rimraf dist && webpack",
```
for Windows compatibility

---

Ah, I knew there had to be some reason to use an npm module 😄